cpg_workflows/workflow.py

```diff
@@ -697,18 +681,18 @@ def _is_reusable(self, expected_out: ExpectedResultT) -> tuple[bool, Path | None
                 Path | None: first missing path, if any
         """
         if self.assume_outputs_exist:
-            logging.info(f'Assuming outputs exist. Expected output is {expected_out}')
+            logging.debug(f'Assuming outputs exist. Expected output is {expected_out}')
```

Comment:
I've set a couple of these messages to be printed only when we use `--verbose` to up the logging level to debug. Reasoning here is that the code as it stands both prints all the paths it intends to check, then the paths which don't exist. This really clouds the logging when resuming runs which have partially completed, when what we really want to know is the paths that were expected but don't exist, which is printed below (line 707).